src/frontend/js/modules/table-of-content.js

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
>src/frontend/js/modules/table-of-content.js

You're using it as a separate class, not a module. Module — is a part initializing by [Module Dispatcher](https://github.com/codex-team/moduleDispatcher).

So:

Don't call it "module". Or  initialize it via `<div data-module"table-of-content"></div>`